### ISSUE TITLE
Can't reproduce paper results on COCO images

### ISSUE THREAD

**Issue by sandpiturtle** [zjhuang22/maskscoring_rcnn#26]
The pretrained network gives strange results for images even from COCO dataset (the predictions are almost random)

<img width="847" alt="Screenshot 2019-03-24 at 18 35 06" src="https://user-images.githubusercontent.com/18504735/54917855-07703f80-4efd-11e9-8439-d3f712bffa56.png">

Looks like the problem is the weights files. 

What I've done:
- Used [predictor.py](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/demo/predictor.py)  and [Mask_R-CNN_demo.ipynb](https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/demo/Mask_R-CNN_demo.ipynb) to generate and visualize masks 
- Used `configs/e2e_ms_rcnn_R_50_FPN_1x.yaml` and `configs/e2e_ms_rcnn_R_101_FPN_1x.yaml`
- Downloaded pretrained models from [here](https://github.com/zjhuang22/maskscoring_rcnn/blob/master/README.md#pretrained-models)

**Comment by sandpiturtle** [comment]
@zjhuang22 Could you please shed some light on the situation? Has the provided checkpoint been initialized with just ImageNet weights or the model has been trained on COCO?

**Comment by fanyix** [comment]
@sandpiturtle I think those are just ImageNet pretrained weights. I cannot find models that are trained on COCO.

**Comment by YubinXie** [comment]
@sandpiturtle How did you load the pre-trained model to Mask_R-CNN_demo.ipynb? It would be great if you can provide details.

I think one possible reason is that, the so called pre-trained model that we want is after the training of author, which gives us the testing accuracy from the paper. While what is more likely is that, the author did not provide the post-training model; instead, the so called pre-trained model is actually raw weight without any segmentation training.

@zjhuang22 Can you confirm if you provided post-training model? Thank you so much!

**Comment by sandpiturtle** [comment]
Ok, I trained the model and now it works as expected. The provided weights file was just initialized with ImageNet weights. Now it is confirmed.

<img width="872" alt="Screenshot 2019-03-28 at 12 11 16" src="https://user-images.githubusercontent.com/18504735/55153514-ebfe7200-5152-11e9-9044-44e22455ae3f.png">

**Comment by YubinXie** [comment]
@sandpiturtle The prediction you provided obviously missed one animal... While the original MASKRCNN captures... Did you run the full training? Can you provide some examples that if scoring works while mask rcnn fails?
